src/StackExchange.Redis/ConfigurationOptions.cs

```diff
@@ -312,6 +312,8 @@ private static RemoteCertificateValidationCallback TrustIssuerCallback(X509Certi
             };
         }
 
+        private static readonly Oid _serverAuthOid = new Oid("1.3.6.1.5.5.7.3.1", "1.3.6.1.5.5.7.3.1");
```

Comment:
Just wondering why not the friendly name parameter is something like serverAuth..? Or else using the single parameter constructor?

Reply:
Good question; in my local testing, this gives the `FriendlyName` of `"TLS Web Server Authentication"` which is perhaps a little confusing since we're not really connection to a "web server", so perhaps `new Oid("1.3.6.1.5.5.7.3.1", "TLS Server Authentication");` might be more sensible. Thoughts? @NickCraver also?

Reply:
I...didn't know that was a thing... Either sounds good here - cheers for the TIL!

Reply:
Mainly "because no one will look at the friendly name".  The single param ctor causes the friendly name to be looked up, and Windows XP through Win 8.1 sometimes caused that to kick off a lookup over to the domain controller... so in the libraries we always use the two parameter version when the Oid instance won't be seen outside the library.